Doc/whatsnew/3.12.rst

```diff
@@ -685,6 +685,9 @@ Removed
   *context* parameter instead.
   (Contributed by Victor Stinner in :gh:`94172`.)
 
+* Namespaces ``typing.io`` and ``typing.re``, deprecated in Python 3.8,
+  are now removed. The items in those namespaces can be imported directly
+  from ``typing``. (Contributed by Sebastian Rittau in :gh:`92871`.)
```

Comment:
3.12!?

Reply:
Sebastian knows this, the PR was initially targeted for 3.12 but we had to retarget to 3.13 because of https://github.com/python/cpython/pull/92873#issuecomment-1273355993.